### PR TITLE
Added a contrib link to the About page and fixed hero 404s.

### PR DESCRIPTION
**Pull request by xenoworf**
It was too easy to miss the side menu's Contribution section, so I added a link on the About page. Users can now get to the Contribution section by clicking on the front page Get Started link, and then the contribution link.

image
image

### CODE REVIEW

**Review by sdadn — APPROVED** [review]

File: src/terra-dev-site/home/HomeHero.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import React from 'react';
-import Particles from 'react-tsparticles';
-import imgTerra from './assets/terra-ui.png';
-import styles from './HomeHero.scss';
+import React from "react";
```
Comment (neilpfeiffer) [inline src/terra-dev-site/home/HomeHero.jsx:1]: Linter expects these to all be single quotes. Not sure why Travis didn't complain unless it's turned off.
Screen Shot 2022-03-04 at 11 48 08 AM

Comment (xenoworf) [reply src/terra-dev-site/home/HomeHero.jsx:1]: I saw so many linter problems that I ignored them and just let "CTRL+S" do the work. I can fix.

File: src/terra-dev-site/about/WhatIsTerra.a.about.mdx [hunk 2]
```diff
@@ -18,3 +19,16 @@ Check out the [Getting Started Guide](/guides/terra-ui/installing-components) to
...
+
+## Contributing
+
+<div className="tui-illustration-grid">
+<div className="tui-illustration-grid-col">
+
+Our [contribution guide](/about/terra-ui/contributing/contribution-guidelines) explains the project's contribution process and standards.
+
+</div>
+<div className="tui-illustration-grid-col tui-illustration-grid-col-illustration">
```
Comment (neilpfeiffer) [inline src/terra-dev-site/about/WhatIsTerra.a.about.mdx:31]: Something is happening with the css (haven't investigated further) but the new image and column is not responsively resizing the same at the top one/existing SVG illustration is, to keep it inline and fully visible (notice reviewing on mobile):
Screen Shot 2022-03-04 at 12 23 34 PM

**Comment by xenoworf** [comment]
Closing for now. I'm too busy to work on this and I'm having problems with the SVG where it's causing errors I don't undertand.

**Pull request closed without merging**